topology/graph/graph.go

```diff
@@ -1058,12 +1064,15 @@ func (g *Graph) String() string {
 
 // MarshalJSON serialize the graph in JSON
 func (g *Graph) MarshalJSON() ([]byte, error) {
+	nodes := g.GetFilteredNodes()
```

Comment:
I think we should use the SubGraph step #417, we could use it as a prefix in the WebUI ?